portfolio/src/main/webapp/html/comments.html

```diff
@@ -10,8 +8,19 @@
   </head>
   <body onLoad="getData()">
     <div id="content">
-      <h1>Fetch Test</h1>
-      <div id="data-container"></div>
+      <h1>POST TEST</h1>
```

Comment:
This header can be a bit intuitive, like "Enter Comment" or the like.

Reply:
done